cli/src/core/parse/packageSpec_test.go

```diff
@@ -0,0 +1,194 @@
+package parse
+
+import (
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"testing"
+
+	"cli/cmd/flags"
+	"cli/core"
+	"cli/core/state"
+
+	"github.com/luno/jettison/jtest"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func copyFile(src, dst string) error {
+	// Open the source file
+	sf, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer sf.Close()
+
+	// Create the destination file
+	df, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer df.Close()
+
+	// Copy the contents of the source file to the destination file
+	_, err = io.Copy(df, sf)
+	if err != nil {
+		return err
+	}
+
+	return nil
+
```

Comment:
```suggestion
```

Reply:
Done in [895254a](https://github.com/openhie/package-starter-kit/pull/34/commits/895254ac521594dec27e0701216486768591e1b7)